src/main/resources/script_templates/ImageJ2-Scripts/Rotate_Stack.py

```diff
@@ -0,0 +1,84 @@
+# @ImageJ ij
+# @Dataset data
+ 
+import math
+ 
+from net.imagej.axis import Axes
+from net.imglib2.util import Intervals
+ 
+from net.imglib2.realtransform import RealViews
+from net.imglib2.realtransform import AffineTransform2D
+from net.imglib2.interpolation.randomaccess import LanczosInterpolatorFactory
+from net.imglib2.view import Views
+ 
+ 
+def get_axis(axis_type):
+    return {
+        'X': Axes.X,
+        'Y': Axes.Y,
+        'Z': Axes.Z,
+        'TIME': Axes.TIME,
+        'CHANNEL': Axes.CHANNEL,
+    }.get(axis_type, Axes.Z)
+     
+ 
+def crop_along_one_axis(ij, data, intervals, axis_type):
+    """Crop along a single axis using Views.
+ 
+    Parameters
+    ----------
+    intervals : List with two values specifying the start and the end of the interval.
+    axis_type : Along which axis to crop. Can be ["X", "Y", "Z", "TIME", "CHANNEL"]
+    """
+ 
+    axis = get_axis(axis_type)
+    interval_start = [data.min(d) if d != data.dimensionIndex(axis) else intervals[0] for d in range(0, data.numDimensions())]
+    interval_end = [data.max(d) if d != data.dimensionIndex(axis) else intervals[1] for d in range(0, data.numDimensions())]
+ 
+    interval = interval_start + interval_end
+    interval = Intervals.createMinMax(*interval)
+ 
+    output = ij.op().run("transform.crop", data, interval, True)
+ 
+    return output
+ 
+angle = 90
+ 
+# Get the center of the images so we do the rotation according to it
+center = [int(round((data.max(d) / 2 + 1))) for d in range(0, data.numDimensions())]
+ 
+# Convert angles to radians
+angle_rad = angle * math.pi / 180
+ 
+# Build the affine transformation
+affine = AffineTransform2D()
+affine.translate([-p for p in center])
+affine.rotate(angle_rad)
+affine.translate(center)
+ 
+# Get the interpolator
+interpolator = LanczosInterpolatorFactory()
+ 
+# Iterate over all frame in the stack
+axis = Axes.TIME
+output = []
+for d in range(data.dimension(axis)):
+ 
```

Comment:
@hadim these are interesting scripts.  One suggestion I have is to make sure they work with either a sample image, or a publicly available image, then provide instructions on which image to use with each script.  

Sometimes it is not obvious that a script needs an image in a certain format (2D, 3D, time series, etc.) so it is nice to direct the user to an image that works, so they can demo the script.